client/src/components/GuestPicker.jsx

```diff
@@ -17,6 +25,40 @@ export default class GuestPicker extends Component {
   componentWillUnmount() {
     document.removeEventListener('click', this.handleOutsideClick, false);
   }
+  handleAddGuests(e, label) {
+    e.preventDefault();
+    const stateLabel = `${label.toLowerCase()}Num`;
+    const otherLabel = stateLabel === 'adultsNum' ? 'childrenNum' : 'adultsNum';
+    let labelNum = this.state[stateLabel];
+    let { guests } = this.state;
+    if (stateLabel === 'infantsNum') {
+      if (this.state.infantsNum < 5) {
+        labelNum += 1;
+      }
+    } else {
+      if (labelNum < (this.props.maxGuests - this.state[otherLabel])) {
+        labelNum += 1;
+      }
+      if (guests < this.props.maxGuests) {
+        guests += 1;
+      }
+    }
+    this.setState({ guests, [stateLabel]: labelNum });
+  }
+  handleMinusGuests(e, label) {
+    e.preventDefault();
+    const stateLabel = `${label.toLowerCase()}Num`;
+    const otherLabel = stateLabel === 'adultsNum' ? 'childrenNum' : 'adultsNum';
+    let labelNum = this.state[stateLabel];
+    let { guests } = this.state;
+    if (labelNum > 0) {
+      labelNum -= 1;
+    }
+    if (label !== 'Infants' && (guests - this.state[otherLabel]) > 0) {
+      guests -= 1;
+    }
+    this.setState({ guests, [stateLabel]: labelNum });
+  }
   handleOutsideClick(e) {
     if (this.props.guestDropDownActive) {
       if (this.dropDownRef.current.contains(e.target)) {
```

Comment:
I see you had to do a outside click to close out the dropdown like I had to do for my popup. Good work!